faucet/templates/faucet/modal.html

```diff
@@ -10,11 +10,15 @@ <h5 class="modal-title">Help fund the faucet!</h5>
       <div class="modal-body">
         <p >
           Transparent donation address:</br>
-          <span class="taddress">tmKBPqa8qqKA7vrGq1AaXHSAr9vqa3GczzK</span>
+          <span class="taddress">tmP5cmDU66W2puHFoqfu8Zqe6M4mpsDk5fS</span>
         </p>
         <p>
-          Shielded donation address:</br>
-          <span class="zaddress">ztbx5DLDxa5ZLFTchHhoPNkKs57QzSyib6UqXpEdy76T1aUdFxJt1w9318Z8DJ73XzbnWHKEZP9Yjg712N5kMmP4QzS9iC9</span>
+          Sprout Shielded donation address:</br>
+          <span class="zaddress">ztSwdDwPhpUZ447YU1BqjxrvutHfu2AyENwUohhTMhnWHreAEHTELhRLvqkARmCSudW1GAcrg58TVaqT7oTH1ohFA7k7V11</span>
+        </p>
+        <p>
+          Sapling Shielded donation address:</br>
+          <span class="zaddress">ztestsapling1603ydy9hg79lv5sv9pm5hn95cngfv4qpd6y54a8wkyejn72jl30a4pfhw8u00p93mu4nj6qxsqg</span>
```

Comment:
@charlieok Same with the sapling addresses

Reply:
is this question still unresolved?

Reply:
Kind of, updated it again so it'll use the existing faucet.testnet.z.cash.

If there is some other, general addresses, we could use those. But I think this is fine for now.